app.js

```diff
@@ -506,6 +512,12 @@ d.run(() => {
 				 * @todo Add description for the function and its params
 				 */
 				function(scope, cb) {
+					if (!appConfig.peers.enabled) {
+						scope.logger.info(
```

Comment:
Is `scope.logger` available? Should it be just `logger`?

Reply:
It is available.